### PR TITLE
gmf csv filename can now be configurated

### PR DESCRIPTION
**Pull request by julsbreakdown**
Enable the admin to pick a default name for the query-result csv file

### CODE REVIEW

File: contribs/gmf/src/directives/displayquerygrid.js [hunk 1]
```diff
@@ -239,6 +239,14 @@ gmf.DisplayquerygridController = function($injector, $scope, ngeoQueryResult,
   this.highlightFeatures_ = new ol.Collection();
   this.highlightFeatureOverlay_.setFeatures(this.highlightFeatures_);
 
+  /**
+   * Filename
+   * @type {string}
+   * @private
+   */
+   this.filename_ = $injector.has('gmfCsvFilename') ?
+   $injector.get('gmfCsvFilename') : 'query-results.csv';
```
Comment (sbrunner) [inline contribs/gmf/src/directives/displayquerygrid.js]: Isn't here an indentation issue?

**Comment by fredj** [comment]
please don't merge a pull request if the travis tests are failing.

**Comment by fredj** [comment]
@julsbreakdown please fix it as soon as possible: all the pull requests created after you commit are blocked.

**Comment by julsbreakdown** [comment]
ok I merged the wrong pr....

**Pull request merged**